### ISSUE TITLE
Delete this repo?

### ISSUE THREAD

**Issue by shntnu** [broadinstitute/FeatureAggregation#1]
@EchteRobert

**Comment by shntnu** [comment]
In favor of https://github.com/broadinstitute/FeatureAggregation_single_cell

**Comment by EchteRobert** [comment]
@shntnu Yes! The issue is that I can't. I think I did something wrong while transferring permissions. Are you able to?

**Comment by shntnu** [comment]
Got it

Will handle it

I've asked BITS to be made admin via Slack

<img width="510" alt="image" src="https://user-images.githubusercontent.com/1210428/156409202-14103269-f1be-434a-98e1-7113093f72f9.png">
